cli/elastic.py

```diff
@@ -66,6 +66,33 @@ def store_files(index: str, files: [str], cfg_name: str):
                 )
             print(result)
 
+def store_bulk(file: str, cfg_name: str):
+    elastic_cfg = util.ctx().cfg_factory().elasticsearch(cfg_name)
+    elastic_client = ccc.elasticsearch.from_cfg(elasticsearch_cfg=elastic_cfg)
+
+    util.existing_file(file)
+    try:
+        meta = get_meta()
+    except RuntimeError:
+        util.warning("Could not read metadata")
+        meta = dict()
+    
+    body = ""
+
+    with open(file) as file_handle:
+        for line in file_handle:
+            json_body = json.loads(line)
+            if "index" not in json_body:
```

Comment:
why? If I properly recall, 'index' denotes the target elasticsearch index. What happens upon the absence of an index? Is there a default index? In other words: what does it mean if there is not 'index'?
Also: why should the presence of 'index' suppress the injection of 'cc-meta'? Maybe this becomes clear after I understand what the absence of 'index' means..

Reply:
The behaviour is because of the structure of a bulk request.
The bulk request consists of multiple documents (json) which all have specific indices (json).
Therefore the cc metadata should only be added if the json is a data document and not a "index" document.
```
{"index":{"_index":"testmachinery","_type":"_doc"}}
{"metadata":{"build_id":"1"},"testrun_id":"demorun-pxjmi","phase":"Succeeded","startTime":"2019-01-09T08:18:52Z","duration":1306,"testsRun":3}
{"index":{"_index":"testmachinery","_type":"_doc"}}
{"metadata":{"build_id":"1","testrun_id":"demorun-pxjmi"},"name":"CreateShootTestDef","phase":"Succeeded","startTime":"2019-01-09T08:19:08Z","duration":727}
...
```
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html